config/routes.rb

```diff
@@ -0,0 +1,70 @@
+Rails.application.routes.draw do
+  # The priority is based upon order of creation: first created -> highest priority.
+  # See how all your routes lay out with "rake routes".
+
+  # You can have the root of your site routed with "root"
+  # root 'welcome#index'
+  root 'products#index'
+  resources :users, :products, :orders, :order_items, :reviews
```

Comment:
`resources` will generate default routes that you may not want, specifically for the `reviews` workflow. Consider being more explicit with your routes.